astrosat_users/migrations/0021_auto_20201015_1626.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='customer',
             name='company_type',
-            field=models.CharField(blank=True, choices=[('SINGLE', 'Single'), ('MULTIPLE', 'Multiple')], max_length=64, null=True, unique=True),
+            field=models.CharField(blank=True, choices=[('NON_PROFIT', 'Non-Profit Organisation'), ('LOCAL_AUTHORITY', 'Local Authority'), ('GOV_AND_EXEC_AGENCIES', 'Government & Executive Agencies'), ('NON_DEPT_PUBLIC_BODY', 'Non Departmental Public Body'), ('PUBLIC_CORP', 'Public Corporation'), ('HEALTH_AND_CARE', 'Health & Care'), ('CHARITY', 'Charity'), ('ACADEMICS_OR_EDUCATION', 'Academics, School or any kind of Education'), ('OTHER', 'Other')], max_length=64, null=True),
```

Comment:
I noticed late I had used the wrong enum in the model and failed to re-run the migration, so this makes sense, was confused at first, thought you meant in the model.